internal/services/gateway/base.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MPL-2.0
+
+package gateway
+
+import (
+	"github.com/microsoft/terraform-provider-fabric/internal/common"
+)
+
+const (
+	ItemName           = "Gateway"
+	ItemTFName         = "gateway"
+	ItemsName          = "Gateways"
+	ItemsTFName        = "gateways"
+	ItemDocsSPNSupport = common.DocsSPNSupported
+	ItemDocsURL        = "https://learn.microsoft.com/en-us/power-bi/guidance/powerbi-implementation-planning-data-gateways"
+)
```

Comment:
Try to avoid localization in the links

```suggestion
	ItemDocsURL        = "https://learn.microsoft.com/power-bi/guidance/powerbi-implementation-planning-data-gateways"
)
```